apps/web-giddh/src/app/models/api-models/DaybookRequest.ts

```diff
@@ -12,6 +12,19 @@ export class DaybookQueryRequest {
     public branchUniqueName: string;
 }
 
+export class ExportBodyRequest {
```

Comment:
Duplicate declaration, consider moving to a common file from where it can be reused